graphql_compiler/tests/test_helpers.py

```diff
@@ -127,7 +128,10 @@ def get_schema():
             uuid: ID
             out_Animal_ParentOf: [Animal]
             in_Animal_ParentOf: [Animal]
+            out_Animal_FriendsWith: [Animal]
+            in_Animal_FriendsWith: [Animal]
             out_Animal_OfSpecies: [Species]
+            out_Animal_Eats: [FoodOrSpecies]
```

Comment:
from a schema perspective, this is confusing -- why would we also track what a particular animal eats when we already track what that animal's species eats?